AudibleApi.Common/ContentLicenseDtoV10.cs

```diff
@@ -31,26 +31,17 @@ private static VoucherDtoV10 DecryptLicenseResponse(ContentLicenseDtoV10 content
             byte[] key = new byte[16];
             byte[] iv = new byte[16];
 
-            using var sha256 = System.Security.Cryptography.SHA256.Create();
-            sha256.ComputeHash(keyComponents);
-            Array.Copy(sha256.Hash, 0, key, 0, 16);
-            Array.Copy(sha256.Hash, 16, iv, 0, 16);
+            var hash = System.Security.Cryptography.SHA256.HashData(keyComponents);
```

Comment:
Dotnet added some nice crypto helper methods since this was first written. Use those instead.

Reply:
Noice